extensions/measurement-tracking/src/panels/PanelMeasurementTableTracking/index.js

```diff
@@ -169,14 +171,22 @@ function _mapMeasurementToDisplay(measurement, index) {
  * @param {*} points
  * @param {*} pixelSpacing
  */
-function _getDisplayText(points, pixelSpacing, seriesNumber, instanceNumber) {
+function _getDisplayText(
```

Comment:
Will be interesting to see how this works for generic groups of SCOORDs / measurements

Reply:
In a later refactor, I would actually want the _getDisplayText function to be part of the defition of the measurement-service level tool schema.